sources/packages/web/src/components/common/EnvironmentBanner.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div v-if="!isProduction" class="environment-banner">
+    <span class="environment-banner-text">{{ bannerText }}</span>
```

Comment:
Was there a reason not to use `v-app-bar` ? We should use the OOTB components unless there is a reason.

Reply:
With vuetify it looks like this. 
![image](https://github.com/user-attachments/assets/97dc7714-3043-4256-92b9-c939586549de)

using 
```vue
<v-app-bar height="40" color="#e3a82b"
      ><v-app-bar-title>{{ bannerText }}</v-app-bar-title></v-app-bar
    >
```